.circleci/config.yml

```diff
@@ -154,6 +154,9 @@ jobs:
 
       - run: bazel test ... --test_tag_filters=-e2e,-examples
 
+      # check size of release.tar.gz
+      - run: ls -l dist/bin/release.tar.gz
```

Comment:
we could save it as a circleci artifact instead - then we could hook up buildsize.org if we care to see the size changes reported in PRs